src/libraries/System.Runtime.Loader/ref/System.Runtime.Loader.cs

```diff
@@ -12,6 +12,12 @@ public static partial class AssemblyExtensions
         public unsafe static bool TryGetRawMetadata(this System.Reflection.Assembly assembly, out byte* blob, out int length) { throw null; }
         public static void ApplyUpdate(Assembly assembly, ReadOnlySpan<byte> metadataDelta, ReadOnlySpan<byte> ilDelta, ReadOnlySpan<byte> pdbDelta) { throw null; }
     }
+    public static partial class MetadataUpdater
+    {
+        public static void ApplyUpdate(Assembly assembly, ReadOnlySpan<byte> metadataDelta, ReadOnlySpan<byte> ilDelta, ReadOnlySpan<byte> pdbDelta) { throw null; }
+        public static string GetCapabilities() { throw null; }
```

Comment:
In the API review we said we weren't going to expose `GetCapabilities` publicly yet. Just the other 2.

Do we need to expose it? If so, can you send a mail to fxdr to get it approved?

Reply:
Will just an email be sufficient or will I need to create another review issue like #51159?  It doesn't make sense to me for one of the APIs in the new MetadataUpdater class be internal.